docs/src/increase-observability/metrics/grid.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Monitor Grid metrics
+sidebarTitle: Grid metrics
+description: Understand how to read metrics for Grid environments.
+---
+
+Grid environments consist of:
+
+* App containers: one or more [app containers](../../configuration/app/_index.md)
+* Service containers: zero or more [service containers](../../configuration/services/_index.md)
+* Worker containers: zero or more [worker instances](../../configuration/app/app-reference.md#workers).
+
+Infrastructure metrics report CPU, RAM, and disk space for app and worker containers
+and CPU and disk space for service containers.
+
+App containers are shown first, with the app name and an image corresponding to the app type.
+Service containers follow next with the same pattern and worker containers are shown last.
+
+You can collapse the graphs by clicking **Hide metrics**.
+The graphs switch to an overview of the average resource utilization for the selected container.
+
+![How app container metrics look when minimized](/images/metrics/app-container-minimized.png "0.65")
```

Comment:
<img width="1370" alt="Screenshot 2022-03-28 at 15 45 00" src="https://user-images.githubusercontent.com/45591272/160411535-d6034b19-3630-4aa5-a495-197cb4ede279.png">

Changing for an image with a label "Service/s"